googlemaps/geocoding.py

```diff
@@ -90,7 +90,10 @@ def reverse_geocode(client, latlng, result_type=None, location_type=None,
     :rtype: list of reverse geocoding results.
     """
 
-    params = {"latlng": convert.latlng(latlng)}
+    if (type(latlng) == str) & (latlng[0] not in ("-+0123456789")):
```

Comment:
Few problems:

- As you pointed out, the type check fails. How about `convert.is_string(latlng)` ?
- `&` should be `and`
- Looks like Place IDs contains alphanumeric chars and dashes, so there's a chance `latlng[0] not in ("-+0123456789")` would fail if the first character in a place ID was a digit. Since the string version of a latlng would contain a comma and a Place ID wouldn't, how about checking for that?